var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -79,15 +79,18 @@ class Openmpi(AutotoolsPackage):
     """
 
     homepage = "http://www.open-mpi.org"
-    url = "https://www.open-mpi.org/software/ompi/v3.0/downloads/openmpi-3.0.0.tar.bz2"
+    url = "https://www.open-mpi.org/software/ompi/v3.1/downloads/openmpi-3.1.1.tar.bz2"
     list_url = "http://www.open-mpi.org/software/ompi/"
 
     # Current
+    version('3.1.1', '493f1db2f75afaab1c8ecba78d2f5aab')  # libmpi.so.40.10.0
```

Comment:
It looks like a previous PR already added this version, but it claims to have a different .so number. Can you rebase/merge and confirm the correct .so version number?